### PR TITLE
Collect OCS-must-gather for getting the remains of trash

### PR DESCRIPTION
**Pull request by yati1998**
Collect must-gather for the following commands:
* rbd trash ls --pool <pool>
* ceph rbd task list
https://issues.redhat.com/browse/RHSTOR-2205

BZ: #2014426

Signed-off-by: yati1998 <ypadia@redhat.com>

### CODE REVIEW

File: must-gather/collection-scripts/gather_ceph_resources [hunk 1]
```diff
@@ -217,6 +218,15 @@ for ns in $namespaces; do
             done
         done
 
+        # collecting trash list for ceph rbd
+        printf "collecting trash list for ceph rbd \n" | tee -a  "${BASE_COLLECTION_PATH}"/gather-ceph-debug.log
+         # Inspecting ceph block pools for ceph rbd
+        blockpools=$(timeout 60 oc get cephblockpools.ceph.rook.io -n openshift-storage -o jsonpath="{range .items[*]}{@.metadata.name}{'\n'}{end}")
+        COMMAND_OUTPUT_FILE=${COMMAND_OUTPUT_DIR}/rbd_trash_ls
+        for bp in $blockpools; do
+            { timeout 120 oc -n "${ns}" exec "${HOSTNAME}"-helper -- bash -c "rbd trash ls --pool $bp" >> "${COMMAND_OUTPUT_FILE}"; } >> "${COMMAND_ERR_OUTPUT_DIR}"/gather-rbd-rbd_trash_ls-debug.log 2>&1 &
```
Comment (Madhu-1) [inline must-gather/collection-scripts/gather_ceph_resources:227]: `gather-rbd-rbd_trash_ls-debug.log` to `gather-rbd_trash_ls-debug.log`

Comment (Madhu-1) [reply must-gather/collection-scripts/gather_ceph_resources]: still am seeing `gather-rbd-rbd_trash_ls-"${bp}"-debug.log` it should be `gather-rbd-trash_ls-"${bp}"-json-debug.log`? sorry  forgot to ask to add json

Comment (yati1998) [reply must-gather/collection-scripts/gather_ceph_resources]: Will update.

**Pull request merged**